tests/vmi_lifecycle_test.go

```diff
@@ -886,7 +886,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
 				supportedCpuModels = libnode.GetSupportedCPUModels(*nodes)
 				if len(supportedCpuModels) < 2 {
-					Skip("need at least 2 supported cpuModels for this test")
+					Fail("need at least 2 supported cpu models for this test")
```

Comment:
I personally like failing fast. As in some cases it can dramatically help identifying why the test failed (you would usually see timeout error. and would need to deep dive into cluster state)

Reply:
If this test is filtered out by CI initially there is no need to fail fast or any troubleshooting :)

Reply:
>If this test is filtered out by CI initially there is no need to fail fast or any troubleshooting :)

That is true but if it is not filtered out it can be difficult to know why they fail...